plasma_framework/test/src/framework/BlockController.test.js

```diff
@@ -123,7 +123,7 @@ contract('BlockController', ([_, other]) => {
         });
     });
 
-    describe('new vault', () => {
+    describe('registerVault', () => {
```

Comment:
ouch, I left my previous comment thinking this is under `VaultRegister` instead of `BlockController`. Sorry then I would just put this under `submitDepositBlock` actually.